### PR TITLE
Migrate dex2jar to Maven Central

### LINKED ISSUE #669: d2j-base-cmd and d2j-jasminMissing POM preventing build
I'm building a project with Soot-Up as a dependency and I'm getting a warning saying 

```
[WARNING] Missing POM for com.github.ThexXTURBOXx.dex2jar:d2j-base-cmd:jar:v61
[WARNING] Missing POM for com.github.ThexXTURBOXx.dex2jar:d2j-jasmin:jar:v61
```

Then I get the error:
```
 Failed to create assembly: Error creating assembly archive jar-with-dependencies: /path/to/.m2/repository/com/github/ThexXTURBOXx/dex2jar/d2j-base-cmd/v61/d2j-base-cmd-v61.jar
```

The file "d2j-base-cmd-v61.jar.lastUpdated" exists at this location. Is there something wrong with this dependency?

### PR DESCRIPTION
**Pull request by ThexXTURBOXx**
This should resolve issues with jitpack.io by making it pretty much obsolete.
Releases are done automatically - the patch version is increased automatically and corresponds to the build number in dex2jar's pipeline.

If you need anything else, feel free to contact or ping me or just open an issue at my repository!

Fixes #669

### CODE REVIEW

**Review by JonasKlauke — APPROVED** [review]
👍

**Comment by oxisto** [comment]
I ended up here following a string of errors and issues because of the missing dex dependency. Unfortunately, there is not yet a release on mvn central of sootup that contains this fix here. Do you foresee a release with this fix any time soon?

**Comment by swissiety** [comment]
yes there will be a release in the near future - in the meantime you could use the latest develop via jitpack.io see https://soot-oss.github.io/SootUp/develop/installation/

**Comment by oxisto** [comment]
> yes there will be a release in the near future - in the meantime you could use the latest develop via jitpack.io see https://soot-oss.github.io/SootUp/develop/installation/

Thanks. I‘ll try that as a temporary workaround. The issue is that I am trying to integrate SootUp into https://github.com/Fraunhofer-AISEC/cpg and we don’t want to force our users to use an additional repo besides Maven central. (We already need to do it for our C++ parsers, but that’s another story).

This should at least unblock development and I probably need to wait for your next release before we merge it into our main.

**Pull request merged**
